amundsen_application/static/css/styles.scss

```diff
@@ -51,3 +51,8 @@ form {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+.my-auto {
```

Comment:
Curious of the name choice. What does `my` stand for? `mg-auto` or `margin-auto` might make more sense.

Edit: Oh I get it... margin-y-auto.

Reply:
Yep.